interface/main/calendar/add_edit_event.php

```diff
@@ -99,6 +104,11 @@
 <?php
 //Gets validation rules from Page Validation list.
 //Note that for technical reasons, we are bypassing the standard validateUsingPageRules() call.
+
+if((!$edit && !$view) ) {
+    $_GET['group'] = 'false';
+    $GLOBALS['enable_group_therapy']=false;
```

Comment:
As noted below, modifying a global setting will cause confusion. Suggest you instead create a local variable instead if you need one.